modules/openapi-generator/src/test/java/org/openapitools/codegen/typescript/typescriptnode/TypeScriptNodeClientCodegenTest.java

```diff
@@ -134,4 +136,42 @@ public void mappedApiImportTest() {
         Assert.assertEquals(codegen.toApiImport("Category"), mappedName);
     }
 
+    @Test(description = "correctly produces imports without import mapping")
+    public void postProcessOperationsWithModelsTestWithoutImportMapping() {
+        final String importName = "../model/pet";
+        Map<String, Object> operations = postProcessOperationsHelper(importName);
+
+        codegen.postProcessOperationsWithModels(operations, Collections.emptyList());
+        List<Map<String, Object>> extractedImports = (List<Map<String, Object>>) operations.get("imports");
+        Assert.assertEquals(extractedImports.get(0).get("filename"), importName);
+    }
+
+    @Test(description = "correctly produces imports with import mapping")
+    public void postProcessOperationsWithModelsTestWithImportMapping() {
+        final String importName = "@namespace/dir/category";
+        Map<String, Object> operations = postProcessOperationsHelper(importName);
+
+        codegen.postProcessOperationsWithModels(operations, Collections.emptyList());
+        List<Map<String, Object>> extractedImports = (List<Map<String, Object>>) operations.get("imports");
+
+        Assert.assertEquals(extractedImports.get(0).get("filename"), importName);
+    }
+
+    private Map<String, Object> postProcessOperationsHelper(String importName) {
```

Comment:
Perhaps just nitpicking - but maybe there could be a better name for this method? Right now it's kind of unclear what its intention is (unless I read the actual method body).